File: sample/UnityCli/Assets/Script/common/module.cs


namespace Common
{
	public class moduleException : System.Exception
	{
		public moduleException(string _err) : base(_err)
		{
		}
	}

	public class Response
	{
	}

	public class IModule
	{
		public IModule()
        {
		}

		public Response rsp = null;
	}
}

File: rpc/test_python/module.py
class moduleException(Exception):
	def __init__(self, err:str) -> None:
		super(moduleException, self).__init__(err)
		self.err = err

class Response(object):
	def __init__(self) -> None:
		pass
	
class imodule(object):
	def __init__(self) -> None:
		self.rsp : Response = None